specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json

```diff
@@ -311,260 +308,21 @@
-          },
-          {
-            "$ref": "#/parameters/SubscriptionIdParameter"
-          },
-          {
-            "$ref": "#/parameters/ResourceGroupNameParameter"
-          },
-          {
-            "$ref": "#/parameters/DomainServiceNameParameter"
-          },
-          {
-            "$ref": "#/parameters/ReplicaSetNameParameter"
-          }
-        ],
-        "responses": {
-          "202": {
-            "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously.",
-            "schema": {
-              "$ref": "#/definitions/ReplicaSet"
-            }
-          },
-          "204": {
-            "description": "HTTP 204 (Not Content) if resource does not exist."
-          }
-        },
-        "x-ms-examples": {
-          "Delete ReplicaSet": {
-            "$ref": "./examples/DeleteReplicaSet.json"
-          }
-        },
-        "x-ms-long-running-operation": true
-      },
-      "patch": {
-        "tags": [
-          "ReplicaSets"
-        ],
-        "operationId": "ReplicaSets_Update",
-        "summary": "Update ReplicaSet (PATCH Resource)",
-        "description": "The Update ReplicaSet operation can be used to update the existing deployment. The update call only supports the properties listed in the PATCH body.",
-        "parameters": [
-          {
-            "$ref": "#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "#/parameters/SubscriptionIdParameter"
-          },
-          {
-            "$ref": "#/parameters/ResourceGroupNameParameter"
-          },
-          {
-            "$ref": "#/parameters/DomainServiceNameParameter"
-          },
-          {
-            "$ref": "#/parameters/ReplicaSetNameParameter"
-          },
-          {
-            "name": "replicaSet",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/ReplicaSet"
-            },
-            "description": "Properties supplied to the Update a ReplicaSet operation."
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "HTTP 200 (OK) if the operation was successful.",
-            "schema": {
-              "$ref": "#/definitions/ReplicaSet"
-            }
-          },
-          "202": {
-            "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously.",
-            "schema": {
-              "$ref": "#/definitions/ReplicaSet"
-            }
-          }
-        },
-        "x-ms-examples": {
-          "Update ReplicaSet": {
-            "$ref": "./examples/UpdateReplicaSet.json"
-          }
-        },
-        "x-ms-long-running-operation": true
-      }
     }
   },
   "definitions": {
     "Resource": {
       "description": "The Resource model definition.",
       "properties": {
         "id": {
-          "readOnly": true,
```

Comment:
From ARM guidance, these properties have to be `readonly`. Please see CI check failures for details.

Reply:
This has been fixed.